.gitignore

```diff
@@ -0,0 +1,7 @@
+.idea/
+annexlang.egg-info/
+build/
+dist/*.egg
+__pycache__
+docs/examples/*.pdf
+docs/examples/*.yml.tex
```

Comment:
```suggestion
```
I'd like to keep these two files, even though they can be computed, for illustration purposes.

Reply:
They aren't checked in currently, that's why I added them to the ignore list. But I'm fine with both.